data/other/logs/package/dpkg.yml

```diff
@@ -0,0 +1 @@
+log_path: /var/log/dpkg.log
```

Comment:
Same as alternatives.log, I'm not sure what we're expecting to find in this file :confused:

Reply:
dpkg provide all manually dpkg packages installed, it could be usefull with some apps installing manually some dpkg.